charts/digma-ng/templates/analytics/deployment.yaml

```diff
@@ -53,7 +53,7 @@ spec:
         - name: GlobalExceptionSettings__ExposeInternalDetails
           value: "true"
         - name: Analytics.Endpoints__Default__Secured
-          value: {{ .Values.analyticsApi.secured | quote }}
+          value: {{ if .Values.analyticsApi.ingress.enabled }}"true"{{ else }}{{ .Values.analyticsApi.secured | quote }}{{ end }}
```

Comment:
if `.Values.analyticsApi.ingress.enabled` then `"true"`? shouldn't it be `"false"`?

Reply:
sure, thanks